src/crypto/DeviceList.js

```diff
@@ -165,6 +282,22 @@ export default class DeviceList {
         return res;
     }
 
+    /**
+     * Get the stored device data for a user, in raw object form
+     *
+     * @param {string} userId the user to get data for
+     *
+     * @return {Object} userId->deviceId->{object} devices, or null if
+     * there is no data for this user.
+     */
+    getRawStoredDevicesForUser(userId) {
+        const devs = this._devices[userId];
+        if (!devs) {
```

Comment:
this feels a bit redundant? just spec it to return undef if there is no data?

Reply:
done